src/main/webapp/app/localci/build-agents/build-agent-summary/build-agent-summary.component.html

```diff
@@ -0,0 +1,100 @@
+<div style="padding-bottom: 60px">
+    <h3 id="build-agents-heading" jhiTranslate="artemisApp.buildAgents.summary"></h3>
+    @if (buildAgents) {
+        <p>{{ buildAgents.length }} online agent(s): {{ currentBuilds }} of {{ buildCapacity }} build jobs running</p>
```

Comment:
Was it a conscious decision not to translate this description?

In previous parts `agents` was translated and at least`of` should probably be translated aswell (probably `build jobs running`  too)?